<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CameraParametersShow</class>
 <widget class="QWidget" name="CameraParametersShow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CameraParametersShow</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QFrame" name="parametersFrame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout" columnstretch="1,1,1,1">
      <item row="0" column="3">
       <widget class="QDoubleSpinBox" name="principalXValue">
        <property name="decimals">
         <number>3</number>
        </property>
        <property name="minimum">
         <double>-1000.000000000000000</double>
        </property>
        <property name="maximum">
         <double>1000.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
       </widget>
      </item>
      <item row="6" column="3">
       <widget class="QDoubleSpinBox" name="k3">
        <property name="decimals">
         <number>3</number>
        </property>
        <property name="minimum">
         <double>-10000.000000000000000</double>
        </property>
        <property name="maximum">
         <double>10000.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="pPointLabel">
        <property name="text">
         <string>Principal point</string>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QDoubleSpinBox" name="focusValue">
        <property name="decimals">
         <number>3</number>
        </property>
        <property name="maximum">
         <double>1000000.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="2">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Radial distortion</string>
        </property>
       </widget>
      </item>
      <item row="4" column="3">
       <widget class="QDoubleSpinBox" name="k1">
        <property name="decimals">
         <number>3</number>
        </property>
        <property name="minimum">
         <double>-10000.000000000000000</double>
        </property>
        <property name="maximum">
         <double>10000.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>X:</string>
        </property>
       </widget>
      </item>
      <item row="5" column="3">
       <widget class="QDoubleSpinBox" name="k2">
        <property name="decimals">
         <number>3</number>
        </property>
        <property name="minimum">
         <double>-10000.000000000000000</double>
        </property>
        <property name="maximum">
         <double>10000.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Focus:</string>
        </property>
       </widget>
      </item>
      <item row="7" column="3">
       <widget class="QPushButton" name="applyCalibrationButton">
        <property name="text">
         <string>Apply</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Y:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QDoubleSpinBox" name="principalYValue">
        <property name="decimals">
         <number>3</number>
        </property>
        <property name="minimum">
         <double>-1000.000000000000000</double>
        </property>
        <property name="maximum">
         <double>1000.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
